docs/design/runtime_processing/runtime_recording.md

```diff
@@ -0,0 +1,387 @@
+# Runtime recording
+
+As explained in [Tracepoint section](../trace_points/index.md), CARET records meta-information at initialization and reduces tracepoint data to as much as possible at runtime.
```

Comment:
> reduces tracepoint data to as much as possible
Maybe we don't need "to" in this case.

Reply:
fixed at https://github.com/hsgwa/CARET_doc/pull/6/files